manifests/install.yaml

```diff
@@ -5647,8 +5661,6 @@ spec:
                     properties:
                       datadog:
                         properties:
-                          address:
```

Comment:
Hmm not sure why this diff is showing for datadog. I rebased upstream master and ran codegen and rebuilt manifests.